### PR TITLE
ft: create update store item component

### PR DESCRIPTION
**Pull request by tolls-3**
## Description
Create a component that helps sellers to edit products int the store
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/W8e8DVf6/32-create-edit-item-page

### CODE REVIEW

**Comment by tolls-3** [comment]
[Create edit item page](https://trello.com/c/W8e8DVf6/32-create-edit-item-page)

File: package.json
Comment (shaunorpen) [inline package.json]: We're adding this, but not using it as far as I can tell?

File: src/App.js
Comment (shaunorpen) [inline src/App.js]: Can we delete commented code before merging?

File: src/App.js
Comment (shaunorpen) [inline src/App.js]: Is this extra white space necessary?

File: src/components/inventory/inventory.js
Comment (shaunorpen) [inline src/components/inventory/inventory.js]: Is this extra white space necessary?

File: src/App.js
Comment (shaunorpen) [inline src/App.js]: Can we create this route using the same syntax as the other routes?

File: src/components/updateItem.js
Comment (shaunorpen) [inline src/components/updateItem.js]: This code looks like it's just for testing? Can it be removed before merging?

File: src/components/updateItem.js
Comment (shaunorpen) [inline src/components/updateItem.js]: Can we remove console.logs before merging?

File: src/components/updateItem.js
Comment (shaunorpen) [inline src/components/updateItem.js]: Can we remove this before merging?

File: src/components/updateItem.js
Comment (shaunorpen) [inline src/components/updateItem.js]: This looks like a hard coded file list, perhaps for testing? Can it be removed before merging?

**Review by shaunorpen — APPROVED** [review]
Approving with questions and suggestions, but nothing that impacts functionality.

**Comment by shaunorpen** [comment]
Oops. Didn't notice this was a draft. Sorry!

File: package.json
Comment (tolls-3) [reply package.json]: I have removed it.

File: src/App.js
Comment (tolls-3) [reply src/App.js]: White space has been removed.

File: src/App.js
Comment (tolls-3) [reply src/App.js]: We have to use it in this format in order to pass the id.

File: src/components/updateItem.js
Comment (tolls-3) [reply src/components/updateItem.js]: Removed

File: src/components/updateItem.js
Comment (tolls-3) [reply src/components/updateItem.js]: Done

File: src/components/updateItem.js
Comment (tolls-3) [reply src/components/updateItem.js]: Done

File: src/components/updateItem.js
Comment (tolls-3) [reply src/components/updateItem.js]: Done.

**Comment by DimejiAre** [comment]
@tolls-3 Have you pulled from develop?

**Review by shaunorpen — APPROVED** [review]

**Pull request merged**
